pure.md

```diff
@@ -0,0 +1,43 @@
+# Pure
+
+### Theme setup
```

Comment:
Why do you have this section here? Any particular reason? I can't see the benefit...